src/utils/formUtils.js

```diff
@@ -113,3 +113,21 @@ export const getEntityTags = props => {
 
   return get(entity, ['tags', 'tagList'], []);
 };
+
+/**
+ * Formats Decorator Value according requirements
+ *
+ * @param {string} currentValue
+ * @param {string} newValue
+ * @param {boolean} isNeedToWrapInQuotes
+ * @return {string} Decorated input value
+ */
+export const formatDecoratorValue = (currentValue, newValue, isNeedToWrapInQuotes) => {
```

Comment:
Consider passing pattern as an arg in this method, it will become more flexible and save you the need to extend the method with other checks if one more kind of decorator value will be introduced.
```suggestion
export const formatDecoratorValue = (currentValue, newValue, pattern, isNeedToWrapInQuotes) => {
```

Reply:
done